### PR TITLE
egl: Add eglGetPlatformDisplay support for Android Platform

### PR DESCRIPTION
**Pull request by renchenglei**
Change-Id: Idc11c560bac7c064217fc59aa097484a165d6f06
Tracked-On: OAM-83377
Signed-off-by: Chenglei Ren <chenglei.ren@intel.com>

### CODE REVIEW

**Comment by renchenglei** [comment]
@tapani & @strassek, could you help take a review of this changes? Thanks a lot!

File: src/egl/main/egldisplay.c [hunk 1]
```diff
@@ -238,7 +238,7 @@ _eglFindDisplay(_EGLPlatformType plat, void *plat_dpy)
          dpy->Platform = plat;
          dpy->PlatformDisplay = plat_dpy;
 
-         /* add to the display list */ 
+         /* add to the display list */
```
Comment (strassek) [inline src/egl/main/egldisplay.c:241]: Separate out the style fixes and submit upstream.

**Review by strassek — COMMENTED** [review]
Could you give us an overview in the patch description for why this change is needed? Why is this change needed now and not before? Why do we need to carry this here and not upstream?

File: src/egl/main/egldisplay.c (hunk 1 above)
Comment (renchenglei) [reply src/egl/main/egldisplay.c:241]: Thanks @strassek, I have submitted the PR to upstream, https://gitlab.freedesktop.org/mesa/mesa/merge_requests/1200

**Comment by strassek** [comment]
Merged with rebase, thanks!

**Pull request closed without merging**